packages/integrations/netlify/src/integration-edge-functions.ts

```diff
@@ -122,7 +122,7 @@ export function netlifyEdgeFunctions({ dist }: NetlifyEdgeFunctionsOptions = {})
 					build: {
 						client: outDir,
 						server: new URL('./.netlify/edge-functions/', config.root),
-						serverEntry: 'entry.mjs',
+						serverEntry: 'entry.js',
```

Comment:
```suggestion
						// Netlify expects .js and will always interpret as ESM
						serverEntry: 'entry.js',
```

Let's add a comment here so we don't regress this again.

Reply:
Good idea!